gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStoragePath.java

```diff
@@ -6,8 +6,8 @@
 import static com.google.gcloud.storage.contrib.nio.CloudStorageUtil.checkNotNullArray;
 import static com.google.gcloud.storage.contrib.nio.CloudStorageUtil.checkPath;
 
-import com.google.appengine.tools.cloudstorage.GcsFilename;
 import com.google.common.collect.UnmodifiableIterator;
+import com.google.gcloud.storage.BlobId;
 
 import java.io.File;
 import java.net.URI;
```

Comment:
Actually in this case the `@see` itself is unnecessary as it is pointing to the super method.